project-base/tests/ShopBundle/Acceptance/acceptance/VatInlineEditCest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ShopBundle\Acceptance\acceptance;
+
+use Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\InlineEditPage;
+use Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\LoginPage;
+use Tests\ShopBundle\Test\Codeception\AcceptanceTester;
+
+class VatInlineEditCest
+{
+    /**
+     * @param \Tests\ShopBundle\Test\Codeception\AcceptanceTester $me
+     * @param \Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\LoginPage $loginPage
+     * @param \Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\InlineEditPage $inlineEditPage
+     */
+    public function testVatEdit(AcceptanceTester $me, LoginPage $loginPage, InlineEditPage $inlineEditPage)
+    {
+        $me->wantTo('vat can be edited via inline edit');
+        $loginPage->loginAsAdmin();
+        $me->amOnPage('/admin/vat/list');
+
+        $inlineEditPage->startInlineEdit(1);
+        $inlineEditPage->changeInputValue(1, 'name', 'test edited');
+        $inlineEditPage->save(1);
+
+        $inlineEditPage->assertSeeInColumn(1, 'name', 'test edited');
+    }
+
+    /**
+     * @param \Tests\ShopBundle\Test\Codeception\AcceptanceTester $me
+     * @param \Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\LoginPage $loginPage
+     * @param \Tests\ShopBundle\Acceptance\acceptance\PageObject\Admin\InlineEditPage $inlineEditPage
+     */
+    public function testVatDeleteAndCreate(AcceptanceTester $me, LoginPage $loginPage, InlineEditPage $inlineEditPage)
```

Comment:
I have only question - why have you joined those two tests?

Reply:
because if you want to delete Vat that is used you need select a new one to replace it which takes some time so the VAT is still in the table which means I need to create a new one that isn't used => can be deleted